src/app/_services/group.service.ts

```diff
@@ -326,7 +328,7 @@ export class GroupService {
      * @returns Observable resolved with { success: boolean, message: string }
      */
     public updateIgPublic(igId: string) {
-        let reqUrl = [this.groupV1, igId, 'admin', 'public'].join('/')
+        let reqUrl = [this.groupV2, igId, 'admin', 'public'].join('/')
```

Comment:
I think we can avoid creating a new `groupV2` var if we use `this.group`. 

https://github.com/ithaka/aiw-ui/blob/872076c34cd17a565aa4df3e79add69b9d0b1c30/src/app/_services/group.service.ts#L27

Reply:
OH!!!!!!!!!!!!

Reply:
Done!